java/Makefile

```diff
@@ -0,0 +1,12 @@
+SNAPSHOT_VERSION = $(shell mvn help:evaluate -Dexpression=project.version -q -DforceStdout 2> /dev/null)
+
+version:
+	@echo $(subst -SNAPSHOT,,$(SNAPSHOT_VERSION))
+
+version-set:
+	@mvn versions:set -DnewVersion=$(NEW_VERSION)-SNAPSHOT
+
+release-prepare:
+	@mvn release:prepare
```

Comment:
This needs to be preceded with a version set command.

Reply:
I don't recall if this pushed though.

Reply:
I think this would solve the problem of immediately creating a commit.

https://maven.apache.org/maven-release/maven-release-plugin/update-versions-mojo.html